src/Analyzers/Core/CodeFixes/CodeFixes.projitems

```diff
@@ -22,6 +22,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)AddObsoleteAttribute\AbstractAddObsoleteAttributeCodeFixProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AddRequiredParentheses\AddRequiredParenthesesCodeFixProvider.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)AliasAmbiguousType\AbstractAliasAmbiguousTypeCodeFixProvider.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)AnalyzerOptionsProviders.cs" />
```

Comment:
Is there a reason why we had this items both in the CompilerExtensions and WorkspaceExtensions layers? I think having it in the former makes the inclusion in the latter redundant. Am I missing something?

Reply:
We didn't have it in both. There are two different files, `AnalyzerOptionsProviders` and `AnalyzerOptionsProvider`

Reply:
Ah, I missed that difference!!